File: recruitment-automation-system-ui/src/app/services/models/application-response.ts
/* tslint:disable */
/* eslint-disable */
import { JobListing } from '../models/job-listing';
export interface ApplicationResponse {
  applicationDate?: string;
  candidateEmail?: string;
  candidateName?: string;
  coverLetter?: string;
  id?: number;
  jobListing?: JobListing;
  resumeUrl?: string;
  status?: string;
}

File: recruitment-automation-system/src/main/java/com/RAS/recruitment_automation_system/application/ApplicationResponse.java
package com.RAS.recruitment_automation_system.application;

import com.RAS.recruitment_automation_system.joblisting.JobListing;
import lombok.*;

import java.sql.Date;
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ApplicationResponse {
    private Integer id;
    private String candidateName;
    private String candidateEmail;
    private String resumeUrl;
    private String coverLetter;
    private String status;
    private Date applicationDate;


}